src/Tutorials/GaussianProcessDataGenerator.cs

```diff
@@ -57,18 +69,26 @@ public void Run()
             var randomFunc = sgp.Sample();
 
             // plotting boilerplate
-            var p1 = new OxyPlot.Series.LineSeries
```

Comment:
Add the oxyplot preprocessor directive like I did in my tutorial to 'comment out' the code. This is also done in the LDA example

Reply:
I think I'll leave the OxyPlot stuff for this completely, since we are going to merge the code at some point; I imagine this will eventually be put into some utility folder.

Reply:
Added a simple plotter with directives